File: app/src/main/java/com/example/foreground/constant/MusicConstants.java
package com.example.foreground.constant;


public class MusicConstants {

    public static final int NOTIFICATION_ID_FOREGROUND_SERVICE = 8466503;
    public static final long DELAY_SHUTDOWN_FOREGROUND_SERVICE = 20000;
    public static final long DELAY_UPDATE_NOTIFICATION_FOREGROUND_SERVICE = 10000;

    public static class ACTION {

        public static final String MAIN_ACTION = "music.action.main";
        public static final String PAUSE_ACTION = "music.action.pause";
        public static final String PLAY_ACTION = "music.action.play";
        public static final String START_ACTION = "music.action.start";
        public static final String STOP_ACTION = "music.action.stop";

    }

    public static class STATE_SERVICE {

        public static final int PREPARE = 30;
        public static final int PLAY = 20;
        public static final int PAUSE = 10;
        public static final int NOT_INIT = 0;


    }

}

File: appK/src/main/java/com/example/foreground/constant/MusicConstants.kt
package com.example.foreground.constant

object MusicConstants {
    const val NOTIFICATION_ID_FOREGROUND_SERVICE = 8466503
    const val DELAY_SHUTDOWN_FOREGROUND_SERVICE: Long = 20000
    const val DELAY_UPDATE_NOTIFICATION_FOREGROUND_SERVICE: Long = 10000

    object ACTION {
        const val MAIN_ACTION = "music.action.main"
        const val PAUSE_ACTION = "music.action.pause"
        const val PLAY_ACTION = "music.action.play"
        const val START_ACTION = "music.action.start"
        const val STOP_ACTION = "music.action.stop"
    }

    object STATE_SERVICE {
        const val PREPARE = 30
        const val PLAY = 20
        const val PAUSE = 10
        const val NOT_INIT = 0
    }
}